pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.airbnb.billow</groupId>
     <artifactId>billow</artifactId>
-    <version>2.21</version>
+    <version>2.25</version>
```

Comment:
Weird version bump is because 2.22-2.24 were used for testing non-`master` features. Bad versioning practice but idk the best way to release test versions in our current set-up